core/modules/system/system.admin.inc

```diff
@@ -256,7 +256,22 @@ function system_themes_admin_form($form, &$form_state, $theme_options) {
  */
 function system_themes_admin_form_submit($form, &$form_state) {
   backdrop_set_message(t('The configuration options have been saved.'));
-  config_set('system.core', 'admin_theme', $form_state['values']['admin_theme']);
+
+  $themes = list_themes();
+  $current_default_theme = config_get('system.core', 'theme_default');
```

Comment:
There are 3 times that the `system.core` config is used here. I think it'd be better to have a single `$config = config('system.core');` then adjust the rest accordingly (e.g. `$config->get(...)`).

Reply:
Well, only 2 of these instances were within the same function, but I have updated the PR for those 2 nevertheless. Thanks 👍